hail/src/main/scala/is/hail/types/encoded/ENDArray.scala

```diff
@@ -1,78 +0,0 @@
-package is.hail.types.encoded
```

Comment:
safe to delete because it wasn't being generated. in the future we'll want to move to legacy folder (I know you know this, but want to clarify in case others read)